odoo/fields.py

```diff
@@ -3551,7 +3551,7 @@ def _add_default_values(self, env, values):
             current_model = env[self.model_name]
             definition_record_field = current_model._fields[self.definition_record]
             container_model_name = definition_record_field.comodel_name
-            container_id = env[container_model_name].browse(container_id)
+            container_id = env[container_model_name].sudo().browse(container_id)
```

Comment:
shouldn't we take the test from the 16.0 branch ?
(or be sure we fully forward port the 16.0 PRs before we merge this one)

Reply:
Yes, I added that commit to pass the tests. We will have to merge the first (ref: https://github.com/odoo/odoo/pull/146265). Then, I will rebase the branch and remove it :smiley: